pkgs/development/python-modules/boto3/default.nix

```
@@ -13,7 +13,7 @@
 
 buildPythonPackage rec {
   pname = "boto3";
-  version = "1.34.58"; # N.B: if you change this, change botocore and awscli to a matching version
+  version = "1.34.109"; # N.B: if you change this, change botocore and awscli to a matching version
```

```suggestion
  inherit (botocore) version; # N.B: if you change this, change botocore and awscli to a matching version
```
Could you please use `botocore.version`?
It helps to avoid version mismatches.